### PR TITLE
Change MDCB upgrade order

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/upgrading-tyk/upgrading-tyk.md [hunk 1]
```diff
@@ -90,13 +90,17 @@ sudo yum update
...
 
-1. Dashboard
-2. Gateway
-3. MDCB
+1. MDCB
+2. Pump
+3. Dashboard
+4. Gateway
+
+Then your Slave DC Gateways in the following order:
```
Comment (letzya) [inline tyk-docs/content/upgrading-tyk/upgrading-tyk.md]: Please change slave to worker

File: tyk-docs/content/upgrading-tyk/upgrading-tyk.md [hunk 2]
```diff
@@ -90,13 +90,17 @@ sudo yum update
...
 Our recommended sequence for upgrading a MDCB installation is as follows:
 
-Master DC:
+Master DC first in the following order:
 
-1. Dashboard
-2. Gateway
-3. MDCB
+1. MDCB
+2. Pump
```
Comment (letzya) [inline tyk-docs/content/upgrading-tyk/upgrading-tyk.md]: Please add `pump (if in use)`

File: tyk-docs/content/upgrading-tyk/upgrading-tyk.md [hunk 3]
```diff
@@ -90,13 +90,17 @@ sudo yum update
...
+3. Dashboard
+4. Gateway
+
+Then your Slave DC Gateways in the following order:
 
-Then your Slave DC Gateways
+1. Pump
```
Comment (letzya) [inline tyk-docs/content/upgrading-tyk/upgrading-tyk.md]: Please add `pump (if in use)`

**Review by letzya — CHANGES_REQUESTED** [review]
Can we add Martin's explanation somewhere? would be good for the users to understand the logic behind.

File: tyk-docs/content/upgrading-tyk/upgrading-tyk.md [hunk 4]
```diff
@@ -90,13 +90,22 @@ sudo yum update
...
+Then your slave DC Gateways in the following order:
+
+1. Pump
 2. Gateway
-3. MDCB
 
-Then your Slave DC Gateways
+We do this to be backwards compatible and upgrading MDCB first followed by the master DC then slave DC Gateways ensures that:
+
+1. It's extremely fast to see if there are connectivity issues, but the way Gateways in slave mode work means they keep working even if disconnected
```
Comment (letzya) [inline tyk-docs/content/upgrading-tyk/upgrading-tyk.md]: slave...

Comment (marksou) [reply tyk-docs/content/upgrading-tyk/upgrading-tyk.md]: That was changed

**Review by letzya — APPROVED** [review]

**Pull request merged**